test/functional/rpc_psbt.py

```
@@ -499,7 +503,151 @@ def test_psbt_input_keys(psbt_input, keys):
+        )
+        assert_equal(psbt2["fee"], psbt3["fee"])
+
+        self.log.info("Test signing inputs that the wallet has keys for but is not watching the scripts")
+        self.nodes[1].createwallet(wallet_name="scriptwatchonly", disable_private_keys=True)
+        watchonly = self.nodes[1].get_wallet_rpc("scriptwatchonly")
+
+        eckey = ECKey()
+        eckey.generate()
+        privkey = bytes_to_wif(eckey.get_bytes())
+
+        desc = descsum_create("wsh(pkh({}))".format(eckey.get_pubkey().get_bytes().hex()))
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)